content/sensu-core/0.29/guides/troubleshooting-sensu.md

```diff
@@ -0,0 +1,322 @@
+
+{{< highlight shell >}}
+tail -n 10000 /var/log/sensu/sensu-server.log > sensu-server-10k.log && gzip -9 sensu-server-10k.log`{{< /highlight >}}
+
+## RabbitMQ Connectivity
+
+In this section, we'll discuss issues faced when connecting to RabbitMQ and how you can go about troubleshooting them.
+
+### Authentication Failures
+
+One of the more common issues that you'll encounter when having RabbitMQ connectivity difficulties is the client and/or server failing to authenticate to RabbitMQ. Let's take a look at what an example error message might look like from both Sensu and from RabbitMQ:
+
+{{< highlight json >}}{
+  "timestamp": "2018-06-25T15:34:54.222674-0500",
+  "level": "warn",
+  "message": "transport connection error",
+  "reason": "possible authentication failure. wrong credentials?",
+  "user": "sensu"
+}{{< /highlight >}}
+
+{{< highlight shell >}}
+tail -f /var/log/rabbitmq/rabbit\@sensu.log
+2018-06-26 01:28:00.439 [info] <0.618.0> accepting AMQP connection <0.618.0> (192.168.1.3:44788 -> 192.168.1.2:5671)
+2018-06-26 01:28:00.442 [error] <0.618.0> Error on AMQP connection <0.618.0> (192.168.1.3:44788 -> 192.168.1.2:5671, state: starting):
+PLAIN login refused: user 'sensu' - invalid credentials
+2018-06-26 01:28:03.469 [info] <0.618.0> closing AMQP connection <0.618.0> (192.168.1.3:44788 -> 192.168.1.2:5671){{< /highlight >}}
+
+As you can see, both RabbitMQ and Sensu will give errors if the credentials are incorrect. We'll now walk through the ways in which you can verify that the credentials are correct.
+
+#### Troubleshooting Authenticating Failures
+
+We'll start by going through the process of setting up RabbitMQ manually. If you've gone through our [RabbitMQ installation guide][4], these commands should be familiar.
+
+- Ensure that you've created the correct vhost:
+
+{{< highlight shell >}}
+sudo rabbitmqctl list_vhosts
+{{< /highlight >}}
+
+This should give you output that looks like:
+
+{{< highlight shell >}}
+Listing vhosts ...
+/
+/sensu{{< /highlight >}}
+
+_NOTE: The `/` in front the `sensu` vhost is required. If you're missing the slash, but have configured Sensu to use the vhost `/sensu`, you will see an error._
+
+If your vhost output doesn't look like the output above, create the vhost:
+
+{{< highlight shell >}}
+sudo rabbitmqctl add_vhost /sensu
+{{< /highlight >}}
+
+- Ensure that the `sensu` user is present:
+
+{{< highlight shell >}}
+sudo rabbitmqctl list_users{{< /highlight >}}
+
+This should give you output that looks like:
+
+{{< highlight shell >}}
+Listing users ...
+sensu   []
+guest   [administrator]{{< /highlight >}}
+
+If the user isn't present, add the user and the password for the user:
+
+{{< highlight shell >}}
+sudo rabbitmqctl add_user sensu secret
+{{< /highlight >}}
+
+_NOTE: If the user is present, and the password needs to be reset, you can reset it by using `sudo rabbitmqctl change_password sensu secret`_
+
+- Ensure that the `sensu` user has the correct permissions for the vhost:
```

Comment:
The `-` makes this indented. Is that intended?

Reply:
Yup, that's part of the styling.